cms/djangoapps/contentstore/views/certificates.py

```diff
@@ -436,6 +441,12 @@ def certificates_detail_handler(request, course_key_string, certificate_id):
 
     store = modulestore()
     if request.method in ('POST', 'PUT'):
+        if certificate_id:
+            active_certificates = CertificateManager.get_certificates(course, only_active=True)
+            if int(certificate_id) in [int(certificate["id"]) for certificate in active_certificates]:
+                # Only global staff (PMs) are able to edit active certificate configuration
+                if not GlobalStaff().has_user(request.user):
```

Comment:
I could be wrong, but I was under the impression that global staff users were the only ones allowed to do *anything*, so could this authorization check simply happen at the top of the method and short-circuit if the user is not GlobalStaff?